webapp/src/services/near-nep21-util.js

```diff
@@ -0,0 +1,325 @@
+import { Contract} from 'near-api-js'
+import getConfig from '../config'
+
+const nearConfig = getConfig(process.env.NODE_ENV || 'development')
+const attachNear1 = '300000000000000';
+const attachNear2 = '60000000000000000000000';
```

Comment:
Let's rename this variables. The first one I guess is `maxGas` the second one is `attachedNear`. Please also make sure that you are using them correctly (in the right order) in functions.